src/steps/generic.rs

```diff
@@ -121,6 +121,14 @@ pub fn run_jetpack(run_type: RunType) -> Result<()> {
     run_type.execute(&jetpack).args(&["global", "update"]).check_run()
 }
 
+pub fn run_rtcl(ctx: &ExecutionContext) -> Result<()> {
+    let rupdate = utils::require("rupdate")?;
+
+    print_separator("rtcl::rupdate");
```

Comment:
Please put a friendlier name

Reply:
Yes, that what's I'm not sure... Do you think in this case `rupdate` or just `rtcl` be a good name?

Reply:
I think rtctl is a better choice

Reply:
Using `rtcl` as separator.